src/Symfony/Component/Emoji/Emojis.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Emoji;
+
+use Symfony\Component\Emoji\Util\GzipStreamWrapper;
+
+/**
+ * @author Simon André <smn.andre@gmail.com>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class Emojis
+{
+    /**
+     * Checks if an emoji exists.
+     */
+    public static function exists(string $emoji): bool
+    {
+        foreach (self::getEmojis() as $value) {
+            if ($emoji === $value) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Returns all available emojis.
+     *
+     * @return iterable<string>
+     */
+    public static function getEmojis(): iterable
+    {
+        $dataFile = __DIR__.'/Resources/data/emoji-en.php';
+        if (!is_file($dataFile) && !is_file($dataFile.'.gz')) {
+            throw new \RuntimeException(sprintf('The emoji data file "%s" does not exist.', $dataFile));
```

Comment:
Is there a hint of want should be done if we want the emoji data file?

Reply:
They shoud be there, as they should be installed by composer with this class  (https://github.com/symfony/emoji/tree/7.1/Resources/data)

Maybe a LogicException would be more accurate ?

Reply:
What purpose does this check serve then anyway?

Reply:
It's been removed